catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -175,18 +204,19 @@ private MetacardImpl initializeMetacard(String id, String contentInput) {
                     .flatMap(Collection::stream)
                     .collect(Collectors.toSet());
 
-            metacard = new MetacardImpl(new MetacardTypeImpl(BasicTypes.BASIC_METACARD.getName(),
-                    BasicTypes.BASIC_METACARD,
+            metacard = new MetacardImpl(new MetacardTypeImpl(metacardType.getName(),
```

Comment:
With the new expanded taxonomy, I'm questioning whether we still need to add these non-normalized attributes to the metacard type.  I'd like to get @jlcsmith's opinion on this.